src/examples/CheckBoxExamplePage.tsx

```diff
@@ -91,6 +92,11 @@ function Example() {
             url:
               'https://github.com/react-native-checkbox/react-native-checkbox',
           },
+          {
```

Comment:
Since the inclusion of a url and wrappedNativeControl=true are always going to be paired you could make the addition of the badge paired to the existence of the url. As in you would pass in {name: 'CheckBox', url: '[real-url-here]'} as the named parameter to Page, and then page would add both the badge and the link. The advantage of having that single parameter is you can't forget one without the other.

Reply:
Updated